pkg/image/containerdregistry/options.go

```diff
@@ -99,7 +100,10 @@ func NewRegistry(options ...RegistryOption) (registry *Registry, err error) {
 		destroy:  destroy,
 		log:      config.Log,
 		resolver: resolver,
-		platform: platforms.Only(platforms.DefaultSpec()),
+		platform: platforms.Ordered(platforms.DefaultSpec(), specs.Platform{
```

Comment:
This is a quick fix for getting this up and running on mac. I think we'll need to eventually match all architectures and unpack to different directories, though I don't know how well this will work for platforms w/ conflicting filesystems (e.g. different endianness)